sphinx_simplepdf/themes/simplepdf_theme/__init__.py

```diff
@@ -14,10 +16,33 @@ def get_html_theme_path():
     return cur_dir
 
 
+def create_custom_css(app):
+
+    def get_config_var(name, default):
```

Comment:
Can we keep the doc_string of the original function here?
I know the code is understandable without it, but it also explains the use case and therefore why it is needed.